includes/abstracts/abstract-wc-legacy-order.php

```diff
@@ -0,0 +1,615 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Legacy Abstract Order
+ *
+ * Legacy and deprecated functions are here to keep the WC_Abstract_Order clean.
+ * This class will be removed in future versions.
+ *
+ * @class       WC_Abstract_Order
```

Comment:
Should be updated to `WC_Abstract_Legacy_Order` (though I noticed we leave this off a lot of classes, and its not a real phpdoc block)